duckduckgo/macos-browser
# Fix main menu/hotkeys in Full Screen video player

Task/Issue URL: https://app.asana.com/0/1177771139624306/1204370242122745/f
https://app.asana.com/0/1177771139624306/1204208308647117/f

**Description**:
- Fixes main menu controls for Full Screen video player
- Fixes closing tab while in Full Screen

**Steps to test this PR**:
1. Open twich.tv, enter full screen, ensure audio can be heard
  - Hit cmd+w - the tab should close stopping the video

2. Open twich.tv, enter full screen
  - Swipe back to the Main Space
  - Select another Tab
  - Swipe to the Space playing the full screen video
  - hit Cmd+W

3. Open twich.tv, enter full screen
  - Swipe back to the Main Space
  - Drag the twich tab to another browser window
  - Select another tab in the target window
  - Swipe to the Space playing the full screen video
  - Hit Cmd+W

4. Open twich.tv, enter full screen
  - Swipe back to the Main Space
  - Drag the twich tab to another browser window
  - Select another tab in the target window
  - Select another tab in the source window
  - Activate another app
  - Swipe to the Space playing the full screen video
  - hit Cmd+W

Test all the Main Menu items while in full screen mode - they should be applied to the video Tab
Test Sharing in full screen




---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
